site/_scss/site/objects/_components.scss

```diff
@@ -0,0 +1,136 @@
+@import 'mixins';
```

Comment:
@amymanion don't import mixins to another partial; instead import it into site/_scss/_styles.scss